src/System.Drawing.Graphics/src/System/Drawing/Graphics/Image.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace System.Drawing.Graphics
+{
+    public enum PixelFormat
```

Comment:
Is there a reason that we're assuming only 32-bit image formats and only these three formats?